packages/api/src/User/User.resolver.ts

```diff
@@ -1,27 +1,42 @@
 import { UseGuards } from '@nestjs/common';
-import { Args, Mutation, Query, Resolver, ResolveField, Parent } from '@nestjs/graphql';
+import {
+  Args,
+  Mutation,
+  Query,
+  Resolver,
+  ResolveField,
+  Parent
+} from '@nestjs/graphql';
 
 import { GQLAuthGuard } from '../Auth/GQLAuth.guard';
 import { User, UserInput } from './User.entity';
 import { UserService } from './User.service';
 import { CurrentUser } from './CurrentUser.decorator';
-import { UserPreferences, UserPreferencesInput } from '../UserPreferences/UserPreferences.entity';
+import {
+  UserPreferences,
+  UserPreferencesInput
+} from '../UserPreferences/UserPreferences.entity';
 import { UserPreferencesService } from '../UserPreferences/UserPreferences.service';
 
 @Resolver(() => User)
 export class UserResolver {
-
   constructor(
     private readonly userService: UserService,
     private readonly userPreferencesService: UserPreferencesService
-  ) { }
+  ) {}
 
   @UseGuards(GQLAuthGuard)
   @Query(() => [User])
   users() {
     return this.userService.findAll();
   }
 
+  @UseGuards(GQLAuthGuard)
+  @Query(() => [User])
+  searchUsers(@Args('param') param: string) {
```

Comment:
Can you rename this to `query`? `param` is too generic

Reply:
Hey @tristanMatthias, "param" has been updated to "query".

Reply:
Did you push? I can't see this yet

Reply:
@LeeYang2019 Did you see the above comment from @tristanMatthias?
Looks like you haven't pushed your updated code yet.